docs/scenarios_authoring/Intro.md

```diff
@@ -13,7 +13,7 @@ Nussknacker provides a drag and drop visual authoring tool (Nussknacker Designer
 &nbsp;
 ## Nussknacker scenario diagram
 
-Nussknacker scenario is a representation of a decision algorithm. The decisions are based on data, which can be anything from clicks on a website, bank transactions, to readings from sensors. These data are processed by Nussknacker, according to the scenario (algorithm). The results generated by Nussknacker are also data; this time they communicate the result or decisions.
+Nussknacker scenario represents a decision algorithm in a form of a graph. The decisions are based on data, which can be anything from clicks on a website, bank transactions, to readings from sensors. These data are processed by Nussknacker, according to the scenario (algorithm). The results generated by Nussknacker are also data; this time they communicate the result or decisions.
```

Comment:
I'm glad to see the "graph" word in the doc :)

But I wonder if talking about "decision algorithm" in general is OK. In the context of RR, it's OK for sure. But in the context of streaming processing ... hmm, it's still OK to talk like that?

WDYT?